README.md

```diff
@@ -0,0 +1,24 @@
+# Py Unittest Walkthrough
+
+This repository is designed to be a learning tool for learning the usage and best practices of the Python unittest package 
+
+
+## Usage 
+
+To see how it works, if you have python3x installed you can run individual tests by:
+
+    cd test
+    cd unit_tests
```

Comment:
Let's instruct `cd test/unit_tests` instead of multiple commands.

Reply:
done